test/packages/other/readme_structure/_dev/build/build.yml

```diff
@@ -0,0 +1,4 @@
+dependencies:
+  ecs:
+    reference: git@1.10
+  docs_structure_enforced: true
```

Comment:
This setting should not be under `dependencies`.

It could also belong to the `validation.yml` file at the root of the package.